package.json

```diff
@@ -11,7 +11,7 @@
     "dashboarding"
   ],
   "private": true,
-  "version": "2.19.0",
+  "version": "2.19.1",
   "branch": "2.x",
```

Comment:
Should the branch be changed to 2.19 as well?

Reply:
I have referred the previous PR's, branch isn't [altered](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5368/files).Same with 2.17.1 increment changes.
Or Is this a different scenario as we would be having upcoming patch releases for 2.19.0

Reply:
the branch here is used for doc site url essentially. so it will just redirect to `https://opensearch.org/docs/2.x/`